src/test/java/com/icegreen/greenmail/imap/commands/SearchTermBuilderTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * #%L
+ * Alfresco Repository
+ * %%
+ * Copyright (C) 2005 - 2020 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package com.icegreen.greenmail.imap.commands;
+
+import java.util.Locale;
+import org.junit.Test;
+
+public class SearchTermBuilderTest {
+
+    @Test
+    public void testCreateDateSearchTermWithNonEnglishLocale()
```

Comment:
Does the test fail if the fix is removed from SearchTermBuilder? Is it worth adding a negative test?

Reply:
Yup, just added a false positive test.